sklearn/__init__.py

```diff
@@ -15,6 +15,78 @@
 import sys
 import re
 import warnings
+import os
+from contextlib import contextmanager as _contextmanager
+
+_ASSUME_FINITE = bool(os.environ.get('SKLEARN_ASSUME_FINITE', False))
```

Comment:
If we want to add more config in the future, don't we want them to be bundled in a dictionary?